core/src/main/scala/org/http4s/util/Renderable.scala

```diff
@@ -60,6 +61,16 @@ object Renderer {
         case Right(b) => rb.render(writer, b)
       }
   }
+
+  implicit val inet4AddressRenderer: Renderer[Inet4Address] = new Renderer[Inet4Address] {
```

Comment:
I don't think we needed these.  And if we do, it should use the method added to util ported from Guava.